test/parallel/test-async-wrap-check-providers.js

```diff
@@ -114,20 +114,30 @@ new ChildProcess();
 
 new HTTPParser(HTTPParser.REQUEST);
 
+{
+  const server = http2.createServer();
+  server.on('stream', common.mustCall((stream) => {
+    stream.respond({ ':status': 200 });
+    stream.end('hello world');
+  }));
+  server.listen(0, () => {
+   const client = http2.connect(`http://localhost:${server.address().port}`);
+   const req = client.request({':path': '/'});
```

Comment:
Style nit: the call to `stream.respond()` on line 120 has padding inside the curly braces but this doesn't.